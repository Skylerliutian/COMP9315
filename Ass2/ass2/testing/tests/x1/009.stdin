N A 10
s A 0
s A 2
s A 4
s A 6
s A 8
O A

i A 0
i A 1
i A 2
i A 3
i A 4
i A 5
i A 6
i A 7
i A 8
i A 9