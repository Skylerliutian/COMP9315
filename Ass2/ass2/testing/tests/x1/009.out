0000000101010101
True
False
True
False
True
False
True
False
True
False
